src/Element_TEST.cc

```diff
@@ -825,6 +825,25 @@ TEST(Element, GetNextElementMultiple)
   ASSERT_EQ(child2->GetNextElement(""), nullptr);
 }
 
+/////////////////////////////////////////////////
+/// Helper function to add child elements without having to create descriptions
+sdf::ElementPtr AddChildElement(sdf::ElementPtr _parent,
```

Comment:
Nit: we usually use `camelCase` for free functions

Reply:
Updated in https://github.com/ignitionrobotics/sdformat/pull/620/commits/d51930495c7946659fd5f74131faf613099b4d05. I'm never sure what to do about free functions and static functions because I think we diverge from the [google style guide](https://google.github.io/styleguide/cppguide.html#Function_Names) on this.

Reply:
Yeah we don't have this documented anywhere, it's one of those things that we need to document